### PR TITLE
organization settings (name change)

### PR DESCRIPTION
**Pull request by elevchyt**
Allows org admins to change the organization's name.

### CODE REVIEW

File: apps/web/src/app/(authenticated)/organization/components/users-table.tsx
Comment (trixobird) [inline apps/web/src/app/(authenticated)/organization/components/users-table.tsx:1]: I think this shouldn't be part of this PR

File: apps/web/src/app/(authenticated)/organization/components/name-edit.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  return (
+    <form
+      onSubmit={form.onSubmit((values) => {
+        handleSubmit(values);
+      })}
+    >
+      <Flex align="center" gap="sm" wrap="wrap">
+        <Text>{t('organizationName')}:</Text>
+        <TextInput placeholder={t('title')} key={form.key('name')} {...form.getInputProps('name')} />
```
[the remark this answers is not in the document]
Comment (elevchyt) [reply apps/web/src/app/(authenticated)/organization/components/name-edit.tsx:43]: Shouldn't be possible to type for non authorized in the first place, fixed via access utils.

File: apps/web/src/app/(authenticated)/organization/components/users-table.tsx
Comment (elevchyt) [reply apps/web/src/app/(authenticated)/organization/components/users-table.tsx:1]: The members list was supposed to be done in this branch but this changed. Let's keep it as it'll be used soon.

File: apps/web/src/app/(authenticated)/organization/components/name-edit.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (trixobird) [reply apps/web/src/app/(authenticated)/organization/components/name-edit.tsx]: That's cool :) Still BE can throw random errors for random reasons. Probably won't happen here so I am resolving this; just keep it in mind.

**Review by trixobird — APPROVED** [review]
you are on fire today :)
